### PR TITLE
[selenium-webdriver] adding typings for remote file detector

### PR DESCRIPTION
**Pull request by ghost**
Please fill in this template.

- [x] Make your PR against the `master` branch.
- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `tsc` without errors.
- [x] Run `npm run lint package-name` if a `tslint.json` is present.  (not present)

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/SeleniumHQ/selenium/blob/d943ee2950e25a2f4a3492058107cfd005d6f7fd/javascript/node/selenium-webdriver/remote/index.js#L556
- [x] Increase the version number in the header if appropriate. (not appropriate)
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "../tslint.json" }`.  (simple changes)

### CODE REVIEW

**Comment by zhengbli** [comment]
@BillArmstrong @Kuniwak @cnishina can you help review this change? Thank you

**Comment by cnishina** [comment]
These changes might be done in the following PR that I'm still working on: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/14454

If this is not a duplicate effort, please let me know and I will review it.

**Comment by ptom-gohealth** [comment]
@cnishina This PR exposes the remote.FileDetector.   This is not done in your PR.

**Comment by ptom-gohealth** [comment]
@cnishina Could you review this?

**Comment by cnishina** [comment]
Do you need this to be in @types/selenium-webdriver 2.53.x or 3.0.x?

**Comment by ptom-gohealth** [comment]
@cnishina 2.53.x

**Comment by cnishina** [comment]
Alright, I'll take a look at this. Might take me a few days to get around to it.

File: selenium-webdriver/remote.d.ts [hunk 1]
```diff
@@ -65,3 +65,25 @@ export class DriverService {
      */
     stop(): webdriver.promise.Promise<any>;
 }
+
```
Comment (cnishina) [inline selenium-webdriver/remote.d.ts]: Should add JSDocs here:

```
/**
 * A {@link webdriver.FileDetector} that may be used when running
 * against a remote
 * [Selenium server](http://selenium-release.storage.googleapis.com/index.html).
 *
 * When a file path on the local machine running this script is entered with
 * {@link webdriver.WebElement#sendKeys WebElement#sendKeys}, this file detector
 * will transfer the specified file to the Selenium server's host; the sendKeys
 * command will be updated to use the transfered file's path.
 *
 * __Note:__ This class depends on a non-standard command supported on the
 * Java Selenium server. The file detector will fail if used with a server that
 * only supports standard WebDriver commands (such as the ChromeDriver).
 *
 * @final
 */
```

**Review by cnishina — CHANGES_REQUESTED** [review]

File: selenium-webdriver/remote.d.ts (hunk 1 above)
Comment (ptom-gohealth) [inline selenium-webdriver/remote.d.ts]: @cnishina pr updated

**Comment by cnishina** [comment]
@andy-ms lgtm!  😄 👍

File: selenium-webdriver/remote.d.ts (hunk 1 above)
Comment (cnishina) [inline selenium-webdriver/remote.d.ts]: thanks!

**Comment by ptom-gohealth** [comment]
@zhengbli this has been reviewed.   can it get merged now?

**Comment by ptom-gohealth** [comment]
@zhengbli @andy-ms This was reviewed a week ago.  Could someone merge it?

**Comment by cnishina** [comment]
@andy-ms Let's get this one merged in. Thanks!

**Comment by cnishina** [comment]
Thanks! @minestarks @zhengbli

**Pull request merged**